### DISCUSSION TITLE
API ed errore CORS

### DISCUSSION THREAD

**Discussion by horsep0wa** [fattureincloud/api#342] [Q&A] [answered]
Buongiorno,
sto interrogando le API attraverso un token personale ma con problemi di CORS. 

In questa fase di sviluppo sto aggirando la problematica eseguendo il browser con `--disable-web-security` ma avrei bisogno di capire meglio in che modo deve essere gestita la situazione:
- esiste un punto in cui è possibile inserire i gli `origin` abilitati?
- come posso effettuare gli sviluppi in locale (tramite localhost e dns anche in http)?

Ho navigato la documentazione ma non mi sembra di aver visto informazioni a riguardo.

Grazie mille

**Comment by valmoz** [comment, marked as answer]
Buongiorno,
le nostre API sono disponibili solo tramite Backend. Non possono essere usate da codice eseguito lato browser.

Buona giornata
Mauro

**Reply by horsep0wa** [reply]
@valmoz chiaro, grazie!

**Reply by valmoz** [reply]
Prego, restiamo a disposizione per ulteriori domande.

Mauro
